tecken/api/views.py

```diff
@@ -711,65 +594,33 @@ def stats(request):
     start_yesterday = start_today - datetime.timedelta(days=1)
     last_30_days = today - datetime.timedelta(days=30)
 
-    if not all_uploads:
```

Comment:
If you had the `can_view_all` permission (which only a handful of us do), then you get the summarized data from the UploadsCreated model. Everyone else who's logged in gets the raw data from the tables. The data from the UploadsCreated model is probably faster to retrieve because it's summarized, but it's only available to a handful of people.

Instead, I'm nixing it and forcing the code to calculate the stats from the raw data for everyone. Counts are problematic, so if this slows down the front page for everyone, then I'll write up a new bug and look at either removing the stats information altogether (who even cares about it?) or cache the answers or something.